### DISCUSSION TITLE
Formatting in c#. Force spreadsheet to numbers.

### DISCUSSION THREAD

**Discussion by DanCadwellcs** [ClosedXML/ClosedXML#2159] [Q&A]
I am trying to set cells to Number.  I have spent 2 days on this Please Help

**Comment by DanCadwellcs** [comment]
Figured it out.  numbers do not have " " around them

**Comment by jahav** [comment]
https://docs.closedxml.io/en/latest/features/cell-format.html#built-in-number-format

What Excel shows in style menu as Number is XLPredefinedFormat.Number.Precision2 I believe.
